tutorials/integration.md

```diff
@@ -345,122 +388,134 @@ for more information see here](https://medium.com/jeremy-keeshin/hello-world-for
 
 These are the steps to start the development:
 
-1. Install browserify so it will be available as a script.
-+	
+1. Install browserify so it will be available as a script.	
 ```bash
 sudo npm install -g browserify
 ```
 1. Create and change to a directory.
 1. Run this command to create the initial package definition file:
-+
 ```bash
 npm init -y
 ```
 1. Install the GSN package and its dependencies:
-+
 ```bash
-npm install @opengsn/gsn@"^2.0.0" ethers
+npm install @opengsn/gsn@"^2.0.3" ethers
 ```
 1. Write your code in a file, for example `index.js`. You can use 
 [`require`](https://nodejs.org/en/knowledge/getting-started/what-is-require/) just as you 
 would with Node.js.
 1. To compile the application into browser-compatible JavaScript, use this command:
-+
 ```bash
 browserify index.js -o bundle.js
 ```
-+
 {% hint style="note" %}
-### NOTE:
+##### NOTE:
 At writing there is a bug that causes the output to have some junk characters. 
-{% endhint %}
 Under Linux you can use the `tr` command to solve this:
-+
 ```bash
 browserify index.js | tr -dc '\0-\177' > bundle.js
```

Comment:
this actually works with 
`browserify index.js -o bundle.js`
with no mangling of character set.

Reply:
In some environments. I still get the error: 
![image](https://user-images.githubusercontent.com/12722969/103103792-2d060d80-45e9-11eb-8780-f2a9fc93b3e4.png)

so this needs to stay

Reply:
I double checked, it still fails

![image](https://user-images.githubusercontent.com/12722969/103158310-f0b9e500-4781-11eb-9731-6b4471d8372c.png)

When the step is done is exactly as you specified
```
#! /bin/bash

browserify etherless.js -o bundle.js
# | tr -dc '\0-\177' > bundle.js
mv bundle.js /var/www/html
cp etherless.html /var/www/html

echo Done at
date
```

Same result

Reply:
AFAIK, latest browserify works, but never mind about it.